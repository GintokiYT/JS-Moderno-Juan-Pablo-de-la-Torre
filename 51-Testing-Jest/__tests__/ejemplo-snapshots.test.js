//* Actualia la snapshot
// npm t -- -u 

const cliente = {
  nombre: 'Juan 2',
  balance: 500,
  tipo: 'Premium',
};

describe('Testing al cliente', () => {
  test('Es Juan Pablo', () => {
    expect(cliente).toMatchSnapshot();
  });
});