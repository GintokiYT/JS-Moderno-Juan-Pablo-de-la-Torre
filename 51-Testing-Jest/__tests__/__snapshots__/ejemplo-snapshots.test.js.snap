// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Juan Pablo 1`] = `
Object {
  "balance": 500,
  "nombre": "Juan 2",
  "tipo": "Premium",
}
`;
